llvm/test/CodeGen/RISCV/GlobalISel/legalizer/rv32/legalize-add.mir

```diff
@@ -181,3 +181,330 @@ body:             |
     PseudoRET implicit $x10, implicit $x11, implicit $x12
 
 ...
+---
```

Comment:
Do you need to add -mattr=+v to these tests?

Reply:
you mean to add additional `-mattr` since it requres `ST.getELen() == 64`?
by the way, is `ST.getELen() = 64` equivalent to `mtriple=riscv64`?

Reply:
ELEN is the vector element length. Zve32x, Zve32f extensions have ELEN==32. Zve64x, Zve64f, Zve64d, and V extensions have ELEN==64